fontc/src/lib.rs

```diff
@@ -3040,14 +3041,23 @@ mod tests {
         let result = TestCompile::compile_source("dspace_rules/Basic.designspace");
         let gsub = result.font().gsub().unwrap();
         let feature_list = gsub.feature_list().unwrap();
-        assert_eq!(feature_list.feature_records().len(), 1);
-        assert_eq!(feature_list.feature_records()[0].feature_tag(), "rvrn");
-        assert!(feature_list.feature_records()[0]
+        assert_eq!(feature_list.feature_records().len(), 2);
```

Comment:
perhaps you forgot to commit the changes to the test source file?

Reply:
hm seems.. likely.